todo-with-kuzzle-and-vueJS/README.fr.md

```diff
@@ -0,0 +1,724 @@
+# Todo MVC Step1
+## Pré-requis
+
+Node.js
+
+npm
+
+## Introduction
+Kuzzle permet de gérer de nombreuses données de manière très simple.
+
+Afin de l'installer, vous pouvez vous rendre [ici](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/).
+
+Dans ce how-to, nous allons réaliser une simple todo-list utilisant 
+le pattern modèle-vue-controleur avec Kuzzle et VueJS.
+
+Cette première partie mettra en avant les fonctionnalités de 
+Kuzzle telles que:
+- la connexion à Kuzzle,
+- la création d'un index,
+- la vérification de l'existence d'un index,
+- la création d'une collection,
+- la récupération d'une liste des collections existantes,
+- la création d'un document,
+- la suppression d'un document,
+- la mise à jour d'un document,
+- la recherche de documents
+
+Les parties `<template>` et `style`, étant liées à VueJS et Vuetify mais
+n'ayant pas d'interaction avec Kuzzle, ne seront pas détaillées dans ce how-to;
+vous pouvez cependant consulter les fichiers concernés parallèlement à 
+la lecture de ce tutoriel.
+
+## Configuation du projet
+Commençons par installer Vue Cli : 
+```
+npm install -g @vue/cli
+```
+
+Nous pouvons maintenant créer un projet :
+```
+vue create todomvc
+```
+Ajoutez manuellement les fonctionnalités suivantes au projet : `router`
+et `vuex`.
+Sélectionnez ensuite '`Eslint + Standard config`', puis '`Lint on save`',
+et enfin '`In dedicated config files`'.
+Vous pouvez maintenant lancer votre projet via la commande :
+```
+npm run serve
+```
+puis vous rendre a l'adresse http://localhost:8080/.
+
+Nous allons maintenant mettre en place le store, qui nous permettra de
+stocker et d'utiliser certaines données devant être globales à notre 
+application.
+
+Pour cela, modifiez l'export du fichier `/src/store.js` de la façon suivante :
+```js
+export default new Vuex.Store({
+  state: {
+    connectedToKuzzle: false,
+    indexName: 'todolists'
+  },
+  mutations: {
+    setConnection(state, value) {
+      state.connectedToKuzzle = value;
+    }
+  }
+});
+```
+
+#### Framework et plugin
+Nous allons utiliser Vuetify pour les templates, 
+ainsi que vue-izitoast pour afficher des notifications.
+
+Afin de les installer lancez les commandes suivantes et
+choisissez la configuration par défaut :
+```
+vue add vuetify
+npm install vue-izitoast --save
+npm install material-design-icons-iconfont -D
```

Comment:
You didn't mention that one earlier.

Also, wouldn't it be better for that how-to to have all these dependencies you mention already pre-installed, make just a quick mention about it without giving too much details, and get on with the how-to itself?

It seems we're spending a lot of time configuring a Vue.js project. I think that readers following a how-to about how to set up a proper MVC model with Vue.js and Kuzzle already know about the basics of Vue.js
And just to be sure that is the case, in the prerequisites section, you should simply state that this how-to assumes that readers know the basics of Vue.js (with a link to their own tutorials)